internal/datastore/common/sql.go

```diff
@@ -247,21 +247,16 @@ func NewPGXExecutor(txSource TxFactory) ExecuteQueryFunc {
 		var tuples []*core.RelationTuple
 		for rows.Next() {
 			nextTuple := &core.RelationTuple{
-				ObjectAndRelation: &core.ObjectAndRelation{},
-				User: &core.User{
-					UserOneof: &core.User_Userset{
-						Userset: &core.ObjectAndRelation{},
-					},
-				},
+				ResourceAndRelation: &core.ObjectAndRelation{},
+				Subject:             &core.ObjectAndRelation{},
```

Comment:
If a `Subject` does also have a `Relation`, which isn't it named `SubjectAndRelation`, or perhaps why not naming `ResourceAndRelation` just `Relation`?

Reply:
The real change would be to break the relation out from both, but that is a major refactor, so I left it using ObjectAndRelation. Since I did so, I wanted to be clear that the relation for the operation is in the first ONR